kleinanzeigen_bot/__init__.py

```diff
@@ -290,8 +290,10 @@ def handle_captcha_if_present(self, captcha_element_id:str, msg:str) -> None:
 
     def delete_ad(self, ad_cfg: Dict[str, Any]) -> bool:
         LOG.info("Deleting ad '%s' if already present...", ad_cfg["title"])
-
+        
+        pause(1000, 2000)
         self.web_open(f"{self.root_url}/m-meine-anzeigen.html")
+        self.web_open(f"{self.root_url}/m-meine-anzeigen.html", timeout=1000, reload_if_already_open=True)
```

Comment:
I think the timeout value is to high, it is in seconds. which means it would wait up to 16 minutes before failing.

Reply:
Sorry yes I realized with a previous post from you it will be in seconds. I will do some tests later with my new offers and existing offers. To find the right settings here means a lot of tests, but I will do and inform you asap.

Reply:
@Tschak77 did you came to any new conclusions regarding a good timeout value?